recipe/meta.yaml

```diff
@@ -1,18 +1,20 @@
 {% set name = "hatchling" %}
 {% set version = "1.8.0" %}
+{% set build = "0" %}
+{% set checksum = "a4f982fdca0717d8c46bfe7b501302f90aaf2a5302845d550b49c8739681feb2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hatchling-{{ version }}.tar.gz
-  sha256: a4f982fdca0717d8c46bfe7b501302f90aaf2a5302845d550b49c8739681feb2
+  sha256: {{ checksum }}
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: {{ build }}
+  skip: True  # [py<37]
   entry_points:
     - hatchling = hatchling.cli:hatchling
 
```

Comment:
Missing about/license_family
Missing about/doc_url

In host and run, `- python >=3.7` should be `- python`
In run, `- importlib-metadata` should be `- importlib-metadata # [py<38]`

Shouldn't we add setuptools and wheel in host?

Could you run the test suite in test?